pkgs/by-name/xc/xcbeautify/package.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchurl, unzip, lib }:
+
+let
+  inherit (lib) getAttr;
+in
+stdenv.mkDerivation rec {
+  pname = "xcbeautify";
+  version = "1.4.0";
+  arch = if stdenv.hostPlatform.isAarch64 then "arm64" else "x86_64";
+
+
+  src = fetchurl {
+    url = "https://github.com/cpisciotta/xcbeautify/releases/download/${version}/xcbeautify-${version}-${arch}-apple-macosx.zip";
+    sha256 = getAttr arch {
+            arm64 = "sha256-4b4mXT5IfNOS8iOrZASDhTrmOehG4mePcoiKxR+IdZk=";
+            x86_64 = "sha256-adEfAK7n3Q/Yd1deyJx7htX7hZaGDztEeBv4z2A0wzg=";
+        };
+    };
+
+  buildInputs = [ unzip ];
+
+  unpackPhase = ''
+    unzip $src
+  '';
```

Comment:
```suggestion
  nativeBuildInputs = [ unzip ];
```

Reply:
Applying this suggestion causes the derivation to fail with : 
```
unpacker appears to have produced no directories
error: builder for '/nix/store/dnl8nanyrn6h9pza39a0j00bpi274qzv-xcbeautify-1.4.0.drv' 
failed with exit code 1;
       last 3 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/cxff104068z4yp9ggckns70nyry6s4rf-xcbeautify-1.4.0-arm64-apple-macosx.zip
       > unpacker appears to have produced no directories
       For full logs, run 'nix log /nix/store/dnl8nanyrn6h9pza39a0j00bpi274qzv-xcbeautify-1.4.0.drv'.
error: build of '/nix/store/3i43r4jb6f3aqacnmfi0mlrznxlab7wj-fastlane-2.217.0.drv', 
'/nix/store/cvjgf9qc1p329f5vdvfwn0w44cj45igm-status-go-0.174.3-c15f9e7-ios.drv', 
'/nix/store/dnl8nanyrn6h9pza39a0j00bpi274qzv-xcbeautify-1.4.0.drv' failed
make: *** [shell] Error 1
```

Reply:
Ah, a zipfile with no top level directory.

You can just ignore this suggestion or alternatively use 
```nix
  nativeBuildInputs = [ unzip ];
  sourceRoot = ".";
```

Reply:
Thanks for explaining the cause of failure with this approach.
I like how the original `unpackPhase ` looked and I'll use that for now.